maps/vgstation.dmm

```diff
@@ -0,0 +1,14 @@
+"a" = (/turf/space,/area)
```

Comment:
If it actually is completely empty, I don't see why not. The only line in the entire codebase that references the file is commented out, so it shouldn't break anything.